recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -29,6 +29,19 @@ CMDLINE_LOGO ?= '${@oe.utils.conditional("DISABLE_RPI_BOOT_LOGO", "1", "logo.nol
 # to enable kernel debugging.
 CMDLINE_DEBUG ?= ""
 
+# Add means to isolate processors for realtime setups
```

Comment:
What do you mean by the above?

Reply:
I could have written this as "Add a configuration variable to isolate processors..." which has the same intent. I am attempting to describe the purpose of the ISOLATED_CPUS variable

Reply:
I see. I would rephrase that as it was confusing when I first read it.

Reply:
Fair point, the change has been made